backend/app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Http\Controllers;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Hash;
+use App\Models\User;
+use Illuminate\Http\Request;
+
+class UserController extends Controller
+{
+    public function index() {
+        return User::all();
+    }
+    public function store(Request $request) {
+
+        $fields = $request->validate(
+            [
+                'username' => 'required|string',
+                'password' => 'required|string|confirmed',
+                'email' => 'required|string|unique:users,email'
+            ]
+            );
```

Comment:
```suggestion
        $fields = $request->validate([
                'username' => 'required|string',
                'password' => 'required|string|confirmed',
                'email' => 'required|string|unique:users,email'
            ]);
```
Instead using string and | for validation use arrays
Ref: https://laravel.com/docs/9.x/validation#quick-writing-the-validation-logic